configs/base/index.js

```diff
@@ -137,6 +137,8 @@ module.exports = {
       "error",
       { object: "document", property: "querySelector" },
       { object: "document", property: "querySelectorAll" },
+      { object: "window", property: "outerWidth" },
```

Comment:
since the reasoning for this PR is that `window.outer*` properties are commonly confused with `window.inner*`, I think this should be explained via a `message`.

```suggestion
      { object: "window", property: "outerWidth", message: "Did you mean to use `innerWidth` ?" },
```

Reply:
Good point